Ginger/GingerCoreCommon/Repository/ApplicationModelLib/POMModelLib/ApplicationPOMModel.cs

```diff
@@ -58,6 +58,37 @@ public ApplicationPOMModel()
         [IsSerializedForLocalRepository]
         public ObservableList<ElementInfo> MappedUIElements = new ObservableList<ElementInfo>();
 
+        //public ObservableList<ElementInfo> mCopiedUnienedList = new ObservableList<ElementInfo>();
+
+        public void PopulateDuplicatedUnienedElementsList()
+        {
+            foreach (ElementInfo EI in MappedUIElements)
+            {
+                EI.ElementGroup = ElementInfo.eElementGroup.Mapped;
+                mCopiedUnienedList.Add(EI);
+            }
+            foreach (ElementInfo EI in UnMappedUIElements)
+            {
+                EI.ElementGroup = ElementInfo.eElementGroup.Unmapped;
+                mCopiedUnienedList.Add(EI);
+            }
+        }
+
+        private ObservableList<ElementInfo> mCopiedUnienedList = new ObservableList<ElementInfo>();
+
+        public ObservableList<ElementInfo> CopiedUnienedList
+        {
+            get
+            {
+                if (mCopiedUnienedList.Count == 0)
+                {
```

Comment:
you need to initilize mCopiedUnienedList as null and then re-create the list only if it is != null and not if it Count == 0